integration_tests/test_dbapi.py

```diff
@@ -47,7 +47,6 @@ def test_select_query(presto_connection):
     rows = cur.fetchall()
     assert len(rows) > 0
     row = rows[0]
-    assert row[0] == "test"
     assert row[2].split("-")[0] == fixtures.PRESTO_VERSION
```

Comment:
- can `fixtures.PRESTO_VERSION` be "latest" here?
- i think you can remove `.split("-")[0]` from here -- switching to dockerhub-provided image means the version won't contain a hyphen

Reply:
Exactly, the value won't "latest" here. Let me fix it.